Sources/MessagesViewController.swift

```diff
@@ -175,6 +176,25 @@ extension MessagesViewController: UICollectionViewDataSource {
 
 }
 
+// MARK: - UITextFieldDelegate Methods
+
+extension MessagesViewController: UITextViewDelegate {
```

Comment:
I understand. I did join the slack room so I will jump into the conversation there.